packages/nimble-components/src/theme-provider/design-token-comments.ts

```diff
@@ -264,5 +264,6 @@ export const comments: { readonly [key in TokenName]: string | null } = {
     elevation2BoxShadow:
         'The box shadow for elevation 2. Used for components such as menus, banners, tooltips, error notifications, and scrolling.',
     elevation3BoxShadow:
-        'The box shadow for elevation 3. Used for components such as dialogs, overlays, and pop-ups.'
+        'The box shadow for elevation 3. Used for components such as dialogs, overlays, and pop-ups.',
+    graphGridlineColor: 'Gridline color for graphs'
```

Comment:
From our [token naming format](https://github.com/ni/nimble/blob/main/packages/nimble-components/CONTRIBUTING.md#token-naming) it looks like this is:
1. element: graph
2. part: gridline
3. state: n/a, i.e. default state
4. token_type: color

which seems reasonable based on the names from the [visual design](https://xd.adobe.com/view/c7f6acf7-ec63-4e6c-9ebe-10cfc4a557fc-92a5/)! 👍